MAPPING.adoc

```diff
@@ -224,13 +224,13 @@ public class Animal {
 }
 ----
 
-Notice that the `Animal` doest not have an @Entity annotation, as it won't be persisted in the database by itself.
+Notice that the `Animal` doesn't have an @Entity annotation, as it won't be persisted in the database by itself.
 
 On the example above, when saving a Dog instance, Animal class' fields are saved too: `name`, `race`, and `age` are saved in a single instance.
 
 ===== @Inheritance
 
-The strategy to work with inheritance with NoSQL, you can active it by adding the @Inheritance annotation to the superclass.
+The strategy to work with inheritance with NoSQL. You can activate it by adding the @Inheritance annotation to the superclass.
```

Comment:
```suggestion
The strategy to work with inheritance with NoSQL can be activated by adding the `@Inheritance` annotation to the superclass.
```

Reply:
> 

Thanks for the suggestion, @dearrudam . Addressed in [e2db5c62c14938f757f05f0f95ee1c875ba9e722](https://github.com/eclipse/jnosql/pull/475/commits/e2db5c62c14938f757f05f0f95ee1c875ba9e722)